wrappers/python/tests/pool/test_create_pool_ledger_config.py

```diff
@@ -1,31 +1,45 @@
-from tests.utils import pool, storage
+import json
+
+from indy import pool
+from tests.utils import pool as pool_utils, storage as storage_utils
 from indy.error import ErrorCode, IndyError
 
 import pytest
 
 
 @pytest.mark.asyncio
 async def test_create_pool_ledger_config_works(cleanup_storage):
-    await pool.create_pool_ledger_config("pool_1")
+    await pool.create_pool_ledger_config(
```

Comment:
Looks like we need to start a discussion about test procedure design.

My vision is the following. This method provides only ok/exception for checking as a public interface. Integration tests should work on this public interface level only.

Sure we can perform additional checks:

1. Try to create config twice and get corresponded exception, but it is a dedicated test case. In rust we have this test case, but it is in the medium cases bunch. So we didn't port it to python.
2. We can try to open pool with this configuration, but it will actually test case for open_pool method. We have this test case in a dedicated file.
3. We can check that files on file system are actually created, but it is testing of non-documented side effects. It can be good unit tests for libindy, but we don't need this test for python wrapper.